pkg/localmetrics/localmetrics.go

```diff
@@ -89,16 +93,28 @@ var (
 
 // UpdateAWSMetrics updates the total AWS Accounts metric every N hours
 func UpdateAWSMetrics(kubeClient kubeclientpkg.Client, hour int) {
-	awsClient, _ := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
+	metricLogger := log.WithValues("Namespace", "aws-account-operator-operator")
+
+	awsClient, err := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
 		SecretName: account.AwsSecretName,
 		NameSpace:  awsv1alpha1.AccountCrNamespace,
 		AwsRegion:  "us-east-1",
 	})
 
+	if err != nil {
+		metricLogger.Error(err, "Failed to get awsClient")
+		return
+	}
+
 	d := time.Duration(hour) * time.Minute
 	for range time.Tick(d) {
-		accountTotal, _ := account.TotalAwsAccounts(awsClient)
-		MetricTotalAWSAccounts.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(accountTotal))
+		accountTotal, err := account.TotalAwsAccounts(awsClient)
+
+		if err != nil {
+			metricLogger.Error(err, fmt.Sprintf("AAO Failed to get total number of AWS accounts: %s", err))
```

Comment:
Remove "AAO" we don't use that and since this error would only been seen in the aws-account-operator error logs its obvious where its coming from :slightly_smiling_face:

Reply:
ack